src/Umbraco.Core/Services/INodeCountService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Umbraco.Cms.Core.Services
+{
+    public interface INodeCountService
+    {
+        public int GetNodeCount(Guid nodeType);
+        public int GetMediaCount();
```

Comment:
```suggestion
        int GetNodeCount(Guid nodeType);
        int GetMediaCount();
```

Public is not needed interfaces :)